monkvision/monkjs
# Feature/fe 221/improve inspections list

ALL CONTRIBUTIONS WILL BE REVIEWED BY AT LEAST ONE OF OUR FRONTEND TEAM MEMBERS.




## Technical description

Added a list of tasks chip wrapped in an horizontal scroll (in inspection read screen), and a new taskRead screen, with different content depending on the status of the task, inside this screen we can:
- Refresh the task
- Relaunch the task(disabled for now)
- Cancel the task(disabled for now)
- Validate the task(disabled for now)
- Copy the task's id
- See the execution time
- See the creation date 
- See the status
- See an illustration of each status.

## Related to



This PR is related to [Improve Inspection list](https://monkvision.atlassian.net/browse/FE-221)

## Context and Motivation 

In case the client wanted to see his tasks details, it is mandatory to have these screens.

## About Tests


Tested on the following devices

- iOS (iPhone 13 pro)
- Android Pixel 3
- Web browser Chrome


Steps to reproduce

1. Go to inspection read (inspection should have tasks)
2. Press the task chip to see it's details




## Screenshots (optional):
- We can access the task from the inspectionRead task chip list.
- Here are all the chips, with their icons (depending on task status).

![image](https://user-images.githubusercontent.com/50136109/145995036-e5ebdb22-7324-4821-9b83-5185ffe12f7c.png)

![image](https://user-images.githubusercontent.com/50136109/145994249-32b90bc4-363a-4843-9cc8-4d766e44c151.png)

- Task screen of each status:

Not started status:

![image](https://user-images.githubusercontent.com/50136109/145994413-08ea0984-83b7-4b7a-9a10-b5be58a55e23.png)

Todo status:

![image](https://user-images.githubusercontent.com/50136109/145994516-2735d0bc-1705-4075-aebd-008fe0afc35f.png)

In progress status:

![image](https://user-images.githubusercontent.com/50136109/145994642-446b3a60-00a6-4224-9d8d-1f7138954cfa.png)

Done status

![image](https://user-images.githubusercontent.com/50136109/145994730-435aef5a-0963-4832-aa7a-34ff70a94475.png)

Error status:

![image](https://user-images.githubusercontent.com/50136109/145994799-2288614e-6723-4dc6-b64f-80a33a8dde5a.png)

Validated status:

![image](https://user-images.githubusercontent.com/50136109/145994971-03793322-40e4-4ed2-9325-dcaaa7558d9b.png)


*This Pull Request template has been written and generated by Monk JS repository.*
